File: demo-basic-auth-db/src/main/java/com/example/springsecurity/controller/BookController.java
package com.example.springsecurity.controller;

import com.example.springsecurity.model.Book;
import org.springframework.validation.annotation.Validated;
import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RestController;

import java.math.BigDecimal;
import java.util.ArrayList;
import java.util.Arrays;
import java.util.List;

@RestController
@Validated
public class BookController {

    @GetMapping("/books")
    List<Book> findAll() {
        List<Book> livros = new ArrayList<Book>();
        livros.addAll(
                Arrays.asList(
                        new Book("Livro 1", new BigDecimal(1)),
                        new Book("Livro 2", new BigDecimal(21))
                ));
        return livros;
    }

    @GetMapping("/books2")
    List<Book> findAll2() {
        List<Book> livros = new ArrayList<Book>();
        livros.addAll(
                Arrays.asList(
                        new Book("Livro 2", new BigDecimal(1))
                ));
        return livros;
    }

}

File: demo-kotlin-auth/src/main/kotlin/com/example/demokotlinauth/controller/BookController.kt
package com.example.demokotlinauth.controller

import com.example.demokotlinauth.model.Book
import org.springframework.web.bind.annotation.GetMapping
import org.springframework.web.bind.annotation.RestController
import java.math.BigDecimal

@RestController
class BookController {

    @GetMapping("/books")
    fun findAll(): List<Book> {
        return listOf(
                Book("The lord of the rings", BigDecimal(8)),
                Book("The lord of the rings 2", BigDecimal(5))
        );
    }
}